keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -563,6 +579,34 @@ For example, if I maintain `example.com/my-signer`, I can use the following stra
   canary object first, and assess the health of the canary workloads.
 * Once I am satisfied that the change is safe, I edit the live object.
 
+### Publishing the kube-apiserver Trust Bundle
+
+Today, the trust bundle that allows verifying kube-apiserver serving certificate(s)
+at its internal endpoints is distributed into every namespace using a configMap.
+This is so that it can be mounted along with the ServiceAccount token in order
+for the workloads to be able to communicate with the kube-apiserver.
+
+To avoid the need for these configMaps in the future, a ClusterTrustBundle API
```

Comment:
these are effectively an API now, so I think we'll continue to generate them, but we can eventually switch the auto-injected tokens to use a cluster trust bundle to ease what kubelets have to watch

Reply:
Hm, that's right. It'd be nice if we were able to remove them in a similar manner as the SA token secrets, but tracking the actual use of these CMs' content is probably impossible.